.nvmrc

```diff
@@ -1 +1 @@
-16.15.0
+18.14.0
```

Comment:
Update is probably necessary also here https://vscode.dev/github/trezor/trezor-suite/blob/80d45b1adaf5758a138a2b79f3b7ca67351871dc/docker/ci-base/Dockerfile#L100